app/styles/shims/_all.less

```diff
@@ -79,7 +79,6 @@
 // These are the colors defined by the Upfluence Visual Design Guidelines.
 // --------------------------------------------------
 
-@upf-primary-bright-purple: #0d0de6;
```

Comment:
Looks like it has all been replaced w/ `color-primary-500` 🥳 

https://github.com/search?q=org%3Aupfluence+upf-primary-bright-purple++NOT+is%3Aarchived&type=code